src/tbstack.c

```diff
@@ -90,6 +92,22 @@ static void parse_pid_arg(const char *prog, char *arg)
     if (tstr != NULL) {
```

Comment:
BTW I'd return here if `tstr` is `NULL` to save indent level :)

Reply:
Fixed in separate commit: https://github.com/tbricks/tbstack/pull/24/commits/c120509ef6df7130ba6ebc4caa367eb1fd70031f